### ISSUE TITLE
Response type name issue

### ISSUE THREAD

**Issue by ravindrajopo** [ozkeisar/mockingbird#10]
Hello @oz-keisar-d, Ravindra here.  I have the following graphql query and looks like the response type has an issue.  I am not a graphql developer by any means, trying to help backend developer to resolve this. 

![image](https://github.com/ozkeisar/mockingbird/assets/140174861/1586def5-a9e8-4c91-b5e6-13f4578e7546)

When I execute the query by going to the url, the playground shows empty on both sides

![image](https://github.com/ozkeisar/mockingbird/assets/140174861/5a75c283-e15d-4900-877f-fdf7a2a7cec2)

**Comment by ozkeisar** [comment]
You need to remove the square brackets in the response type name field.

In the playground you can see all your existing schemes after clicking on the SECHEMA button in the right.

**Comment by ravindrajopo** [comment]
I removed the [] from the response type, now the playground now showing the query in the left side of the playground like above image
![image](https://github.com/ozkeisar/mockingbird/assets/140174861/9badeb87-c0db-4d9e-982a-82f02e7adc96)

**Comment by ozkeisar** [comment]
You need to write your query, it's always start empty

**Comment by ravindrajopo** [comment]
Hoo I see, Sorry about that.  I don't write graphQL queries, will ask backend dev for some help.  In REST it comfortably brings the mock data but in GraphQL it looks like we need to again write query in playground.

**Comment by ozkeisar** [comment]
Yes I'm working on button that will let you copy generated query, hope it will be in the next version.

**Comment by ozkeisar** [comment]
Version 1.3.0 released
I added the copy query button + fixed the support of response type name to support array type

hope its want you wanted

**Comment by ravindrajopo** [comment]
Hi,  Thanks for taking the effort to add the button.  Trying to locate the COPY QUERY button

**Comment by ravindrajopo** [comment]
I managed to find the copy button.  When copied the query the execution doesn't return mock data
![image](https://github.com/ozkeisar/mockingbird/assets/140174861/d217d623-e08f-4131-850b-8019de765bbc)

Mock response data
![image](https://github.com/ozkeisar/mockingbird/assets/140174861/f2f41663-d2d1-499c-a147-58942b7fffe1)

**Comment by ravindrajopo** [comment]
Need to speak to the BE developer

**Comment by ravindrajopo** [comment]
Closing this ticket as we have the copy button
